tanner/tests/test_mysql_db_helper.py

```diff
@@ -0,0 +1,73 @@
+import unittest
+import asyncio
+import os
+from unittest import mock
+from tanner.utils.mysql_db_helper import MySQLDBHelper
+
+
+def mock_config(section, value):
+    if section == 'SQLI' and value == 'host':
+        return '127.0.0.1'
+    if section == 'SQLI' and value == 'user':
+        return 'root'
+    if section == 'SQLI' and value == 'password':
+        return ''
+
+
+class TestMySQLDBHelper(unittest.TestCase):
+    def setUp(self):
+        self.loop = asyncio.new_event_loop()
+        asyncio.set_event_loop(self.loop)
+        self.filename = '/tmp/db/test_db'
+        os.makedirs(os.path.dirname(self.filename), exist_ok=True)
+        open('/tmp/db/test_db', 'a').close()
+
+        self.db_name = 'test_db'
+        self.expected_result = None
+        self.returned_result = None
+        self.query_map = None
+        self.handler = MySQLDBHelper()
+
+    @mock.patch('tanner.config.TannerConfig.get', side_effect=mock_config)
+    def test_check_db_exists(self, m):
```

Comment:
I'm not sure what you're doing but you should do
1) Make a DB and test it exists
2) when no db present test method returns same result

Reply:
I did almost similar to this, but let me clear all this.